hexa/workspaces/models.py

```diff
@@ -43,18 +47,15 @@ class WorkspaceQuerySet(BaseQuerySet):
     def filter_for_user(
         self, user: typing.Union[AnonymousUser, User]
     ) -> models.QuerySet:
-        return self._filter_for_user_and_query_object(user, Q(members=user))
+        return self._filter_for_user_and_query_object(
+            user, Q(workspacemembership__user=user)
+        )
 
 
 class Workspace(Base):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     name = models.TextField()
     description = models.TextField(blank=True)
-    members = models.ManyToManyField(
```

Comment:
Why did you remove this ?

Reply:
I think this is exactly like the `Team` & `Membership` models. 
The relation is a M2M so I think it's better represented by a `ManytoManyField`

@pvanliefland Do you have an opinion on this ?

Reply:
You are both right ;) 

Ideally you should use a many-to-many field with an intermediate model like explain here: https://docs.djangoproject.com/en/4.1/ref/models/fields/#django.db.models.ManyToManyField.through.

This is indeed what we did for `Team` and `Membership`

Reply:
Okay. I'll revert it.